server.php

```diff
@@ -30,6 +62,11 @@ function show_valet_404()
     parse_url($_SERVER['REQUEST_URI'], PHP_URL_PATH)
 );
 
+/**
+ * Filter host to support xip.io feature
+ */
+$_SERVER['HTTP_HOST'] = support_xip_io($_SERVER['HTTP_HOST']);
+
 $siteName = basename(
     $_SERVER['HTTP_HOST'],
```

Comment:
This should be:

`support_xip_io($_SERVER['HTTP_HOST'])`

...for the reasons mentioned above.